cookbook/frontend/bower.rst

```diff
@@ -43,6 +43,12 @@ create a ``.bowerrc`` file with a new destination (like ``web/assets/vendor``):
         "directory": "web/assets/vendor/"
     }
 
+.. tip::
+
+    If you're using a frontend build system like `Gulp`_ or `Grunt`_, then
```

Comment:
committing assets totally defeats the purpose but is better than not saying anything. so :+1: @weaverryan, but would be nice a more clear message that you can also use npm alone and, despite of SO, successfully!